targets/UnrealMarketplacePlugin/makeCpp.js

```diff
@@ -660,6 +661,20 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
     return "";
 }
 
+function getCustomApiLogic(tabbing, api, apiCall, isInstanceApi) {
+    if (apiCall.name === "ExecuteFunction")
```

Comment:
I don't think this is a strict enough check. api.name and apiCall.name would be specific enough.